configs/components/cfpropertylist.rb

```diff
@@ -1,7 +1,7 @@
 component "cfpropertylist" do |pkg, settings, platform|
   pkg.version "2.2.7"
   pkg.md5sum "be2fc619084422cc28cb68f6084bea91"
-  pkg.url "http://buildsources.delivery.puppetlabs.net/CFPropertyList-2.2.7.tar.gz"
+  pkg.url "https://rubygems.org/downloads/CFPropertyList-#{pkg.get_version}.gem"
```

Comment:
We're not carrying the gem; we've got the tarball up on `buildsources`. I'm trying this with a couple different URLs but this broke the build when I tested it.